lpc55_sign/src/cert.rs

```diff
@@ -26,6 +27,23 @@ pub fn read_certs(paths: &[PathBuf]) -> Result<Vec<Certificate>, Error> {
     Ok(certs)
 }
 
+pub fn read_rsa_private_key(path: &PathBuf) -> Result<RsaPrivateKey, Error> {
+    let bytes = std::fs::read(path)?;
+    if bytes.starts_with("-----BEGIN".as_bytes()) {
+        let (label, der) = pem_rfc7468::decode_vec(&bytes)?;
+        if label == "PRIVATE KEY" {
```

Comment:
This isn't you, but, I'm surprised `pem_rfc7468` assumes the label is UTF-8. The RFC says,

> The ABNF
   in [Section 3](https://www.rfc-editor.org/rfc/rfc7468#section-3) is specific to US-ASCII.  As these textual encodings can
   be used on many different systems as well as on long-term archival
   storage media such as paper or engravings, an implementer ought to
   use the spirit rather than the letter of the rules when generating or
   parsing these formats in environments that are not strictly limited
   to US-ASCII.